src/styles/_base.scss

```diff
@@ -1,6 +1,14 @@
-@import "standard";
 @import "uswds-colors/colors";
 
+/* General styles */
```

Comment:
sass allows one-line comments

Reply:
It does, but sass will keep `/* */` comments in the output but delete `//` comments.

Reply:
ah, very nice, I hadn't looked at that